packages/lablgtk3/lablgtk3.3.1.3/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+
+synopsis: "OCaml interface to GTK+3"
+description: """
+OCaml interface to GTK+3
+
+See https://garrigue.github.io/lablgtk/ for more information.
+"""
+
+maintainer: "garrigue@math.nagoya-u.ac.jp"
+authors: ["Jacques Garrigue et al., Nagoya University"]
+homepage: "https://github.com/garrigue/lablgtk"
+bug-reports: "https://github.com/garrigue/lablgtk/issues"
+dev-repo: "git+https://github.com/garrigue/lablgtk.git"
+license: "LGPL with linking exception"
```

Comment:
```suggestion
license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
```

Reply:
Done. Also added the other related packages, fixing the license there too.